neo/Ledger/Blockchain.cs

```diff
@@ -290,7 +290,7 @@ private RelayResultReason OnNewBlock(Block block)
             {
                 Block block_persist = block;
                 List<Block> blocksToPersistList = new List<Block>();
-
+                                
```

Comment:
remove extra white space

Reply:
EHauheauea, coincidentally we did it together.

Reply:
In fact, I removed the line.